lib/src/room.dart

```diff
@@ -281,32 +304,39 @@ class Room {
     return t is String ? t : '';
   }
 
-  /// The avatar of the room if set by a participant.
+  /// The avatar of the room if set by a participant. If you want to also
+  /// let the avatar be calculated by the room heroes for DM rooms, use
+  /// `Room.calcAvatar()` instead.
   Uri? get avatar {
```

Comment:
I think it is a bad idea to change the behaviour of an existing function and I also don't really like, that this now makes the obvious function name the usually wrong one to use. If I want the room avatar, I will call room.avatar, I don't want to know about the subtleties of calculated avatars vs the avatar state event....

Reply:
What would you suggest?

Reply:
But Avatar is now a Future and a Future should not be returned from a getter IMO. A getter is for something which is directly present. A Future is something we want to fetch, so this needs an operation to be done which is better displayed by a method(). However we would deprecate this avatar getter and mention the new method